scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER,
+  dob DATE,
+  specialty CHARACTER VARYING
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER,
+  dob DATE,
+  insurance CHARACTER VARYING
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date DATE,
```

Comment:
Additionally, I would probably avoid using `date` and `time` as column headings, since they're also data types and SQL is (sometimes) case-insensitive.